pkg/ccloudv2/org.go

```diff
@@ -29,7 +29,8 @@ func (c *Client) CreateOrgEnvironment(environment orgv2.OrgV2Environment) (orgv2
 }
 
 func (c *Client) GetOrgEnvironment(envId string) (orgv2.OrgV2Environment, error) {
-	res, httpResp, err := c.OrgClient.EnvironmentsOrgV2Api.GetOrgV2Environment(c.orgApiContext(), envId).Execute()
+	req := c.OrgClient.EnvironmentsOrgV2Api.GetOrgV2Environment(c.orgApiContext(), envId)
+	res, httpResp, err := c.OrgClient.EnvironmentsOrgV2Api.GetOrgV2EnvironmentExecute(req) // calling Execute() instead will cause mock calls to fail
```

Comment:
Why would mock fail? Does the mock method not exist in the package?

Reply:
Yup, the `Execute()` method isn't mocked.

Only the API methods themselves are mocked, but not the request struct methods.

Reply:
oh right.. i think that's the default way we write those methods so think we can leave out this comment